cli/ballerina-cli/src/main/resources/cli-help/ballerina-test.help

```diff
@@ -100,6 +100,9 @@ OPTIONS
            Include the dependencies and artifacts in the  executable JAR file(s) of a 
            Ballerina program that are required to enable the observability functionalities on it.
 
+       --dump-build-time
```

Comment:
We shouldn't add descriptions for hidden flags. Related to https://github.com/ballerina-platform/ballerina-lang/pull/33932/files#r754899702

Reply:
Removed `--dump-build-time` from help.